.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
This pre-commit hook does not work on Windows. All repositories are tested on cross-platform including Windows.

Reply:
but works in my windows.

![image](https://github.com/hexojs/hexo-front-matter/assets/12471057/82c2bdbd-5f85-442a-b019-3ed5418d1480)

Reply:
successful commit
![image](https://github.com/hexojs/hexo-front-matter/assets/12471057/81f60465-4607-4be5-8f59-768c9940fa20)

Reply:
btw iam currently using `node` v14 and `lint-staged & husky` latest version.

Reply:
- disabled auto init husky https://github.com/dimaslanjaka/hexo-front-matter/commit/a5fbaa3e6f7221e6a2e3a5558a0692e58ee17e73

we can manually enable or disable auto linter.

- drop `prettier`

remove prettier from `lint-staged` which still unsupported

Reply:
check i've re-edit my PR descriptions